.circleci/config.yml

```diff
@@ -26,9 +26,14 @@ jobs:
       - deploy:
           name: Publish charts
           command: |
-            if [ "${CIRCLE_BRANCH}" == "master" ]; then
-              curl -sL https://raw.githubusercontent.com/bitnami/test-infra/master/circle/helm-publish-charts.sh | bash -
-            fi
+            case ${CIRCLE_BRANCH} in
+              "master" | "azure" )
```

Comment:
while this is fine, I wonder if the correct way of doing this making the change in the corresponding branch.